### ISSUE TITLE
[Screen Reader-Project-Makecode Documentation page]: Alt Attribute is not defined for the image showing near Documentation link on Project page.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2800]
**User Experience:**
Users who depends on Screen Reader will get confuse if Alt attribute is not defined for the interactive images, then information about the image will get missed from users.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool Used: IE11 Developer (F12)

**Repro Steps:-**
1- Navigate to https://makecode.microbit.org/projects
2- Navigate to the image showing near Documentation link in left panel.
3- Verify that alt attribute is defined or not for the image.

**Actual Result:-**
Alt Attribute is not defined for the image showing near Documentation link on Project page.

**Expected Result:-**
Alt Attribute should be defined for the image showing near Documentation link on Project page.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Refer Attachment:-**
![mas1 1 1_makecode_alt text is not defined for the image](https://user-images.githubusercontent.com/30434417/29664153-252a22f8-88ec-11e7-99e6-aadd0cdcc51d.png)

**Comment by veler** [comment]
Issue fixed but not deployed yet.

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/projects and issue is fixed now. Hence closing the bug.
